### ISSUE TITLE
Syntax error on python 3.7.0

### ISSUE THREAD

**Issue by cserb** [onfido/api-python-client#5]
I get the following with python 3.7.0 (3.6.6 works fine):

```
Python 3.7.0 (default, Jun 28 2018, 13:15:42) 
[GCC 7.2.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onfido
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cris/.conda/envs/tge_dev/lib/python3.7/site-packages/onfido/__init__.py", line 28, in <module>
    from .models.address import Address
  File "/home/cris/.conda/envs/tge_dev/lib/python3.7/site-packages/onfido/models/__init__.py", line 32, in <module>
    from .check_creation_request import CheckCreationRequest
  File "/home/cris/.conda/envs/tge_dev/lib/python3.7/site-packages/onfido/models/check_creation_request.py", line 35
    def __init__(self, type=None, redirect_uri=None, reports=None, report_type_groups=None, criminal_history_report_details=None, tags=None, suppress_form_emails=None, charge_applicant_for_check=None, brand_id=None, async=None):
                                                                                                                                                                                                                            ^
SyntaxError: invalid syntax
```

**Comment by orf** [comment]
I believe this is related to https://github.com/swagger-api/swagger-codegen/issues/8328. When a release is made with the fix we can fix this 👍

**Comment by asychev** [comment]
Bump

**Comment by QuestofIranon** [comment]
also encountered this

**Comment by ciaran16** [comment]
Hi all, a fix for this will be released in the next couple of weeks.

**Comment by ciaran16** [comment]
This is fixed in 3.0.0
